extensions/ql-vscode/src/contextual/definitions.ts

```diff
@@ -25,27 +29,46 @@ const SELECT_QUERY_NAME = '#select';
 enum KeyType {
   DefinitionQuery = 'DefinitionQuery',
   ReferenceQuery = 'ReferenceQuery',
+  PrintAstQuery = 'PrintAstQuery',
 }
 
 function tagOfKeyType(keyType: KeyType): string {
   switch (keyType) {
-    case KeyType.DefinitionQuery: return 'ide-contextual-queries/local-definitions';
-    case KeyType.ReferenceQuery: return 'ide-contextual-queries/local-references';
+    case KeyType.DefinitionQuery:
```

Comment:
It seems like a bunch of these definitions should be hoisted out of `definitions.ts` into some more general file in `contextual/` but that needn't happen in this PR.

Reply:
Sure. I agree that it's getting a bit messy. I'll work on that next.